pkg/controller.v1/tensorflow/controller.go

```diff
@@ -429,6 +436,14 @@ func (tc *TFController) reconcileTFJobs(tfjob *tfv1.TFJob) error {
 				tfjob.Status.ReplicaStatuses[rtype].Active = 0
 			}
 		}
+
+		//If the job is terminated and its sub-resources are deleted, it no longer need reconciliation,
```

Comment:
I don't think this is correct
Because even a job is failed or succeeded,
there is a cleanupTFJob and it might haven't pass  the TTL, so the tfjob haven't been deleted and need to wait until future reconcile.
If we mark it as finish reconcile here, then there will be no reconcile in the future
the cleanupTFJob will never be triggered